exasol_script_languages_developer_sandbox/templates/vm_bucket_cloudformation.jinja.yaml

```diff
@@ -9,6 +9,44 @@ Resources:
         RestrictPublicBuckets: true
     DeletionPolicy: Retain
     UpdateReplacePolicy : Retain
+  {{role_name}}:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+            - Effect: Allow
+              Principal:
+                  Service:
+                    - vmie.amazonaws.com
+              Action:
+                - sts:AssumeRole
+      Description: !Sub "IAM Role to deploy VM's on ${AWS::StackName}"
```

Comment:
Isn't that the role for VM import/export, so deploy is probably not the best word

Reply:
fixed